scripts/build-images

```diff
@@ -41,15 +41,15 @@ EOF
 if [ "${GOARCH}" != "s390x" ]; then
 xargs -n1 -t docker image pull --quiet << EOF > build/images-cilium.txt
     ${REGISTRY}/rancher/mirrored-cilium-certgen:v0.2.0
-    ${REGISTRY}/rancher/mirrored-cilium-cilium:v1.16.0
+    ${REGISTRY}/rancher/mirrored-cilium-cilium:v1.16.1
     ${REGISTRY}/rancher/mirrored-cilium-cilium-envoy:v1.29.7-39a2a56bbd5b3a591f69dbca51d3e30ef97e0e51
-    ${REGISTRY}/rancher/mirrored-cilium-clustermesh-apiserver:v1.16.0
-    ${REGISTRY}/rancher/mirrored-cilium-hubble-relay:v1.16.0
+    ${REGISTRY}/rancher/mirrored-cilium-clustermesh-apiserver:v1.16.1
+    ${REGISTRY}/rancher/mirrored-cilium-hubble-relay:v1.16.1
     ${REGISTRY}/rancher/mirrored-cilium-hubble-ui:v0.13.1
     ${REGISTRY}/rancher/mirrored-cilium-hubble-ui-backend:v0.13.1
-    ${REGISTRY}/rancher/mirrored-cilium-operator-aws:v1.16.0
-    ${REGISTRY}/rancher/mirrored-cilium-operator-azure:v1.16.0
-    ${REGISTRY}/rancher/mirrored-cilium-operator-generic:v1.16.0
+    ${REGISTRY}/rancher/mirrored-cilium-operator-aws:v1.16.1
+    ${REGISTRY}/rancher/mirrored-cilium-operator-azure:v1.16.1
+    ${REGISTRY}/rancher/mirrored-cilium-operator-generic:v1.16.1
     ${REGISTRY}/rancher/hardened-cni-plugins:v1.4.1-build20240325
```

Comment:
could you bump hardened-cni-plugins to `v1.5.1-build20240805` at the same time?

Reply:
We need to change it also on the chart

Reply:
https://github.com/rancher/rke2-charts/pull/500

Reply:
done

Reply:
Thanks!